api/test/model/test_play_asset_pack_states.gd

```diff
@@ -0,0 +1,23 @@
+# ##############################################################################
+#
+#	Copyright 2020 Google LLC
+#
+#	Licensed under the Apache License, Version 2.0 (the "License");
+#	you may not use this file except in compliance with the License.
+#	You may obtain a copy of the License at
+#
+#		https://www.apache.org/licenses/LICENSE-2.0
+#
+#	Unless required by applicable law or agreed to in writing, software
+#	distributed under the License is distributed on an "AS IS" BASIS,
+#	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#	See the License for the specific language governing permissions and
+#	limitations under the License.
+#
+# ##############################################################################
+extends "res://test/test_helper/base_test_class.gd"
+
+func test_play_asset_pack_states_valid():
```

Comment:
Right now since assertions are made by helper functions like here and here. Test cases don't need to reference the dictionary values again.